### PR TITLE
Adding modal to experiment page. This allows the user to view experim…

### PR DESCRIPTION
**Pull request by UAMS16**
…ent index without having to leave the page

image
image

### CODE REVIEW

File: cegs_portal/search/templates/search/v1/experiment.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
 
+{% load waffle_tags %}
```
Comment (tncowart) [inline cegs_portal/search/templates/search/v1/experiment.html:3]: You aren't using a flag here, you don't need to load waffle_tags.

File: cegs_portal/search/templates/search/v1/partials/_experiment_index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+</script>
+<script>
+    function closeModal() {
+        let modal = document.getElementById("exp-modal");
+        modal.classList.add("closing") // Kicks off an animation on the modal dialog
+        modal.addEventListener("animationend", (event) => {
+            modal.remove(); // remove the dialog from the DOM once the animation is over
+        })
+    }
+</script>
```
Comment (tncowart) [inline cegs_portal/search/templates/search/v1/partials/_experiment_index.html:75]: Do you need this here or in experiment.html? I feel like it should be only in one place.

File: cegs_portal/search/templates/search/v1/partials/_experiment_index.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+<script type="module">
+    import { facetFilterSetup } from "{% static 'search/js/experiment_list/facet_filter.js' %}";
+    import { addDragListeners, addDropListeners, addSelectListeners } from "{% static 'search/js/experiment_list/drag_drop.js' %}";
+    facetFilterSetup();
+    addDropListeners();
+    addDragListeners();
+    addSelectListeners();
+</script>
```
Comment (tncowart) [inline cegs_portal/search/templates/search/v1/partials/_experiment_index.html:83]: I don't think you need the drag and drop set up for the modal.

File: cegs_portal/search/templates/search/v1/partials/_experiment_index.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+{% if logged_in %}
+<script type="module">
+    import { downloadDataSetup } from "{% static 'search/js/experiment_list/download_data.js' %}";
+    downloadDataSetup("{{ csrf_token }}");
+</script>
+{% endif %}
```
Comment (tncowart) [inline cegs_portal/search/templates/search/v1/partials/_experiment_index.html:90]: I don't think users should be able to download stuff from the modal. I think this means you don't need to pass "logged_in" to the template from the view.

File: cegs_portal/search/templates/search/v1/partials/_experiment_index.html [hunk 5]
```diff
@@ -0,0 +1,91 @@
+{% load static i18n %}
+
+{% block content %}
```
Comment (tncowart) [inline cegs_portal/search/templates/search/v1/partials/_experiment_index.html:3]: This is a template, so it doesn't extend base.html and thus, doesn't have a content or inline_javascript block.

**Review by tncowart — CHANGES_REQUESTED** [review]

File: cegs_portal/search/templates/search/v1/partials/_feature_sig_reg_effects.html [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {% load humanize %}
 {% load custom_helpers %}
 <div id="modal">
-<div class="modal-underlay" onclick="closeModal()"></div>
```
[the remark this answers is not in the document]
Comment (UAMS16) [reply cegs_portal/search/templates/search/v1/partials/_feature_sig_reg_effects.html:4]: This change was an accidental deletion being undone. This file shouldn't have been changed at all.

[the remark this answers is not in the document]
Comment (tncowart) [reply cegs_portal/search/templates/search/v1/partials/_feature_sig_reg_effects.html:4]: This still hasn't been resolved.

**Review by tncowart — CHANGES_REQUESTED** [review]

**Review by tncowart — APPROVED** [review]

**Pull request merged**